apps/src/studio/studio.js

```diff
@@ -1709,6 +1709,12 @@ Studio.reset = function(first) {
   Studio.sayComplete = 0;
   Studio.playSoundCount = 0;
 
+  // More things used to validate level completion.
+  Studio.removedItemCount = 0;
+  Studio.setActivityRecord = null;
+  Studio.hasSetBot = false;
+  Studio.hasAddedItem = false;
```

Comment:
In general, I think it would be nice if we could avoid adding more and more items to the Studio namespace, as it gets pretty crowded. I wonder if we could namespace this a little bit better. Maybe something like Studio.behaviorTracking.removedItemCount, etc. (or a better name than behaviorTracking)